.travis.yml

```diff
@@ -86,6 +86,7 @@ addons:
     - intltool
 before_script:
 - pip install PyYAML
+- pip install ansible
```

Comment:
Hi, Boris asked me why it's failing, so commenting here.
I'm clueless about this project, may speak nonsense, feel free to ignore :)

My main suspicion is does this `pip` install for same python version as `python3` is later running?  Maybe it should be `pip3`?
I recommend writing `python3 -m pip install ...` to be more confident it matches.

Instead of installing specific packages here, consider keeping the list of dependencies with the code, e.g. travis_ci/requirements.txt and telling pip to use that.  Shouldn't matter, just more idiomatic.

(You could go further than that into using a venv for isolation, at which point it's worth adopting Poetry or Pipenv.  But then you won't benefit from system packages like `python-gobject` installed above, and Travis builds are disposable anyway, so I don't think you want this.)

Reply:
than you very much Beni! actually adding the pip install ansible actually changed the error and now it is "gi" it is missing, I am just trying to figure out how not to make the tests run even longer by installing more packages, my feeling was that some of the tests all ready use osinfo.py and I am not adding anything that is not used there, so I was thinking I am not adding the test in the right place or something...

Reply:
Ah, I see the `No module named 'ansible'` was in an earlier build, seems `pip install ansible` did help :tada:.  I still recommend `python3 -m pip install` form.